pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -266,15 +361,23 @@ def state(self):
         @property
         def create_ops_list(self):
             """Returns create_ops_list function of the matching precision."""
-            return create_ops_listC64 if self.use_csingle else create_ops_listC128
+            if not self._mpi:
+                return create_ops_listC64 if self.use_csingle else create_ops_listC128
+            return create_ops_listMPIC64 if self.use_csingle else create_ops_listMPIC128
 
         @property
         def measurements(self):
             """Returns Measurements constructor of the matching precision."""
+            if not self._mpi:
```

Comment:
It is preferable to invert the direction of the logic here, starting with ``if self._mpi``

Reply:
Done!